com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -1,22 +1,48 @@
 {
   "app-id": "com.jetbrains.IntelliJ-IDEA-Community",
   "command": "idea",
-  "runtime": "org.freedesktop.Sdk",
-  "runtime-version": "18.08",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
   "separate-locales": false,
-  "sdk": "org.freedesktop.Sdk",
-  "finish-args": [
-    "--device=all",
-    "--device=dri",
-    "--env=IDEA_JDK=/app/idea-IC/jre64/",
-    "--env=IDEA_VM_OPTIONS=/app/idea-IC/bin/idea64.vmoptions",
-    "--filesystem=host",
-    "--share=ipc",
-    "--share=network",
-    "--socket=x11",
-    "--talk-name=org.freedesktop.Notifications",
-    "--talk-name=org.freedesktop.secrets"
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
```

Comment:
I think we had this one clear: when using `"sdk": "org.freedesktop.Sdk"`, there is no need to bundle any SDK, you can install them individually, at any time, and the same will be available at `/usr/lib/sdk/`. Can you explain the reason why would we need these bundled?

I'll tell you my point of view. There are tons of SDKs, and there are tons of versions. For Java, there are currently 4 versions. Imagine if you have to do the same for `Node.js`, `Go`, `Erlang`...and all the others that are supported.

Reply:
> I think we had this one clear: when using "sdk": "org.freedesktop.Sdk", there is no need to bundle any SDK, you can install them individually, at any time, and the same will be available at /usr/lib/sdk/. 
I think if you are referring to we meaning you and I, not really clear as far as your explanation of how that is the case with respect to Sdk vs Platform, and how that makes the install of the appropriate libraries. I have say that I haven't tried to use the ide flatpak since Installed the extensions onto my copy of Silverblue, because I was running my fork. I see your point about the support of all the sdk's, and take it as a reason to know more about the (feature) you mention above.